packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -100,7 +100,7 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       menuIconProps && menuIconProps.className,
       isPrimaryButtonDisabled!,
       checked!,
-      !!this.state.menuProps && !this.props.split
+      !!this.state.menuProps && (!this.props.split || !!this.props.splitButtonFullActiveButtonStyling)
```

Comment:
I'm not sure this prop is a good abstraction here. I'd like to see us always pass expanded as true through to getClassNames when the menu is open. Would it work to update getClassNames to take in an isSplit parameter and use that to prevent the styling of the primary button for the default fabric visuals?

Reply:
Yes!! tried the change and that works, more cleaner. Let me update the PR.